SC Evo Complete/SCEvo_Core.SC2Mod/Base.SC2Data/GameData/SCBW/Protoss_Units.xml

```diff
@@ -4752,14 +4744,20 @@
         </CardLayouts>
         <Radius value="0.125"/>
         <SeparationRadius value="0.125"/>
-        <SubgroupPriority value="19"/>
-        <MinimapRadius value="0"/>
-        <AIEvalFactor value="0"/>
-        <Mass value="0.2"/>
+        <MinimapRadius value="0.125"/>
         <GlossaryPriority value="180"/>
         <CostResource index="Minerals" value="15"/>
         <BuildTime value="6.5625"/>
-        <GlossaryCategory value=""/>
+        <Deceleration value="2"/>
+        <LateralAcceleration value="115"/>
+        <Sight value="5"/>
+        <InnerRadius value="0.125"/>
+        <ScoreMake value="15"/>
+        <AINotifyEffect value="AIPurificationNovaDanger"/>
+        <AIEvaluateAlias value="DisruptorPhased"/>
+        <PushPriority value="9"/>
+        <DefaultAcquireLevel value="Defensive"/>
+        <Response value="Flee"/>
```

Comment:
using a "Visible" target filter in the behavior validator would break when escaping up a ramp, so it's better to make the scarab passive (so it only attacks the target from the Issue Order effect, and anything that happens after that is basically a graphical issue)